src/EditorFeatures/CSharpTest/GenerateFromMembers/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersTests.cs

```diff
@@ -159,7 +159,7 @@ public Program(int i)
         this.i = i;
     }
 
-    public Program(int i, string s, bool b = default(bool)) : this(i)
+    public Program(int i, string s, bool b = false) : this(i)
```

Comment:
Change our current code to use a common codepath, which led to this change.  The old behavior was not desirable, and the new behavior is.